<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LLA_XYZ_Component</class>
 <widget class="QWidget" name="LLA_XYZ_Component">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Longitude</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_lon"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_x">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Latitude</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_lat"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_y"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Altitude</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_alt"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineEdit_z"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
